### DISCUSSION TITLE
Ability to use image volumes with no boxes for training as negatives

### DISCUSSION THREAD

**Discussion by AceMcAwesome77** [Project-MONAI/MONAI#6792] [Q&A]
Hi, I think it would be an important improvement of the MONAI detector models to have the ability to include "negative" image volumes, i.e. image volumes with no pathology bounding boxes, in the training dataset.  The current RetinaNet 3D code for example requires each image volume to have at least 1 bounding box.  There are many real-world examples where we would need negatives to train an accurate model.  For example, if training a model to detect blood clots in a particular artery, you would also need to include image volumes showing that artery without any clot - otherwise, if you only trained on images of clotted arteries at that location, the detector would just classify that artery location as positive whether it was clotted or not.  

I made some minor changes to the Retinanet 3D code that accomplished this, but there may be a more efficient way than mine so I did not branch any code.  I have expanded on this issue in more detail here:

https://github.com/Project-MONAI/tutorials/issues/1292

Thanks!

**Comment by KumoLiu** [comment]
Hi @AceMcAwesome77,
- We have already added a transform to handle empty boxes as training data.
   https://github.com/Project-MONAI/MONAI/pull/6170
- Also updated in the tutorial, https://github.com/Project-MONAI/tutorials/pull/1256.
- You can also refer to this discussion: https://github.com/Project-MONAI/tutorials/discussions/1197.

I hope it helps, thanks! (convert to discussion for now)
